### PR TITLE
A couple changes to prepare for k8s deployment

### PR DESCRIPTION
**Pull request by maeve-fpf**
These are all basically the same things we changed on STN. We now have `DJANGO_LOG_CONSOLE` as an env var. DO doesn't have that set, so this could be deployed there.

### CODE REVIEW

**Comment by maeve-fpf** [comment]
```
gyp verb check python checking for Python executable "python2" in the PATH
gyp verb `which` failed Error: not found: python2
```

:cry:

**Comment by maeve-fpf** [comment]
Added a possibly controversial change: don't uppercase variables that are not settings (https://docs.djangoproject.com/en/3.0/ref/settings). I think this makes it clearer what is there for Django to read and what is our own internal logic to figure out what to set.

File: securedrop/settings/base.py [hunk 1]
```diff
@@ -283,56 +295,75 @@
...
+        'formatter': 'json_out'
+    }
+else:
+    console_log = True
+    log_handler = 'console'
+    log_handlers['console'] = {
+        'level': log_level,
+        'class': 'logging.StreamHandler',
+        'formatter': 'json_out',
+        'stream': sys.stdout,
```
Comment (chigby) [inline securedrop/settings/base.py:335]: This line needs an `import sys` at the top of the file

File: devops/docker/NodeDockerfile [hunk 2]
```diff
@@ -1,13 +1,11 @@
-ARG NODE_VER
-ARG NPM_VER
-FROM node:${NODE_VER}
+# sha256 as of 2020-06-30 for node:10-alpine
+FROM node@sha256:bafda8e474ba8973f146cb3fc2d531cf7fb7fec73ef4ca04aa38f26db520a883
```
Comment (chigby) [inline devops/docker/NodeDockerfile:2]: In changing this to alpine linux, it's possible we lost access to `python` and `make`, which we need to build `node-sass`. It's possible that what is needed here is something akin to what is being done in the production docker file: https://github.com/freedomofpress/securedrop.org/blob/ed18480a3934215685ed0047fc41df102819a1b9/devops/docker/ProdDjangoDockerfile#L11-L13

Comment (maeve-fpf) [reply devops/docker/NodeDockerfile:2]: Thanks! Testing these out now.

**Comment by maeve-fpf** [comment]
Made some more updates, ready for re-review! This is rebased on requirements update PR so that it will pass CI.

**Comment by eloquence** [comment]
@maeve-fpf Could you give us an update on the status of this PR? Is it ready for the web team's review at this point?

**Comment by maeve-fpf** [comment]
@eloquence There is a remaining CI timeout issue that I need to address. Will mark as blocked now and remove after rebasing and resolving that.

**Comment by maeve-fpf** [comment]
Marking this as unblocked, but not sure what to do about making it reviewable - would splitting it up into multiple PRs be better?

**Comment by eloquence** [comment]
I defer to Harris & team; it might make sense to do a pair review with an infra team member.

**Comment by conorsch** [comment]
Sounds like a great plan. Since @maeve-fpf wrote the changes, I volunteer to co-review, either with @harrislapiroff or someone else from web.

**Comment by maeve-fpf** [comment]
I've rewound this so that it doesn't include whitenoise, since that can be reviewed separately -- available today to pair on review here if needed.

**Comment by maeve-fpf** [comment]
Also marking as blocked by https://github.com/freedomofpress/securedrop.org/pull/723.

**Pull request merged**